marklogic-data-hub-central/ui/e2e/cypress/e2e/mastering/e2eMasteringflow.cy.tsx

```diff
@@ -374,10 +368,12 @@ describe("Validate E2E Mastering Flow", () => {
       });
     });
 
-    browsePage.getClearAllFacetsButton().click();
+    browsePage.clearAllFacets();
     browsePage.waitForSpinnerToDisappear();
     cy.waitForAsyncRequest();
```

Comment:
We don't need these commands since they are already inside the clearAllFacets method